packages/flutter_tools/lib/src/isolated/sdk_web_configuration.dart

```diff
@@ -0,0 +1,46 @@
+// Copyright 2022 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// @dart = 2.8
```

Comment:
Is there any blocker to making this null-safe?

Reply:
Yes, the imports from dwds.

Reply:
Ahh, TIL. cool